bto-autobuilder/dell-bto-autobuilder

```diff
@@ -90,15 +90,17 @@ class Install:
     def __init__(self):
         self.old_state = 'starting progress tracking'
         self.state = None
-        self.progress = progressbar.ProgressBar()
-        self.progress.start()
+        if progressbar is not None:
```

Comment:
Does this work? We can check the `progressbar` from within `sys.modules` if this is failing from undefined use.